### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ogsevko**
- [DEMO LINK](https://ogsevko.github.io/layout_antisnake/)
- [TEST REPORT LINK](https://ogsevko.github.io/layout_antisnake/report/html_report/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,36 @@
 body {
   margin: 0;
 }
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  grid-template-rows: 300px 300px;
```
Comment (germandolia) [inline src/styles/main.scss:8]:
```suggestion

```
No need to style rows here

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+
+  &__item {
+    display: inline-grid;
+    place-items: center;
+
+    min-height: 300px;
```
Comment (germandolia) [inline src/styles/main.scss:14]:
```suggestion
    height: 300px;
```
simple height is also ok here

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,36 @@
...
+  }
+
+  .container__item:nth-child(#{$i}) {
+    background-color: rgb(calc(120% - #{$i} * 20%), 0%, 0%);
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  .container {
+    grid-template-columns: 1fr 1fr 1fr;
```
Comment (germandolia) [inline src/styles/main.scss:34]: use ```repeat``` here

**Comment by germandolia** [comment]
Also, test report link is not working

File: src/styles/main.scss (hunk 3 above)
Comment (ogsevko) [reply src/styles/main.scss:34]: Yup! Thanks for you review!

**Pull request closed without merging**